docs/architecture/adr-0024-metrics-scripting.md

```diff
@@ -0,0 +1,113 @@
+# ADR 0024 -- Metrics Scripting
+## Context
+
+We'd like to be able to script metrics reporting to third party consumers of 
+Raster Foundry. Currently we manually assemble metrics when people ask for 
+them. However, enough time has passed and we have consistent enough obligations 
+that we know a few that would be scriptable. This ADR seeks to answer the 
+question of how we ought to script production of a particular subset of 
+metrics, specifically, tile server requests by user.
+
+## Options
+
+This ADR considers three possible solutions and weighs them on extensibility, 
+queryability, platform dependence, and difficulty.
+
+One option is to take our existing infrastructure for metrics collection, move 
+it into a collection of AWS Lambda functions, and orchestrate those Lambda 
+functions with a Step Function. We have a plan for how this would work already 
+(see issues #684 through #688 for the beginnings of how this would work). 
```

Comment:
Links to these issues need to be full URLs (they're in a private repo - which I think is ok). By default github will link to issues in the same repository.